Add include for `root_top.conf` in the nginx.conf

kroegerama opened the pull request:
This pull request allows custom configuration of the root config in the top of the file. This can be used to load modules, which is not possible at the end of the config file. There is already a `http_top.conf`, so `root_top.conf` is a logical addition.

The PR NginxProxyManager/docker-nginx-full#28 added the geoip2 module as requested in Issue #46. But currently there is no way to actually load the modules if you cannot mount custom files to `/etc/nginx/modules/` like in the Truenas Scale NPM App.

Now you can just add a `root_top.conf` to the persistent mounted `/data/custom` folder and add the following content:
```
load_module /usr/lib/nginx/modules/ngx_http_geoip2_module.so;
load_module /usr/lib/nginx/modules/ngx_stream_geoip2_module.so;
```

jc21 commented:
These custom includes are getting wild.

Can you please update the documentation with this addition as well:
https://nginxproxymanager.com/advanced-config/#custom-nginx-configurations

This file can be found at `docs/src/advanced-config/index.md`

kroegerama commented:
Thank you so much for your fast response! I updated the documentation and added a small snippet on how to enable the geoip2 module.
If you don't want the snippet in the documentation, let me know so I can remove it.
